src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -239,21 +244,40 @@ public boolean isNameVisibleTo(FeedbackResponseCommentAttributes comment, Feedba
         if (roster.getStudentForEmail(userEmail) != null) {
             currentUserTeam = roster.getStudentForEmail(userEmail).team;
         }
-        
         for (FeedbackParticipantType type : showNameTo) {
-            if (type == FeedbackParticipantType.GIVER && userEmail.equals(response.giver)) {
-                return true;
-            } else if (type == FeedbackParticipantType.INSTRUCTORS && roster.getInstructorForEmail(userEmail) != null) {
-                return true;
-            } else if (type == FeedbackParticipantType.RECEIVER && userEmail.equals(response.recipient)) {
-                return true;
-            } else if (type == FeedbackParticipantType.OWN_TEAM_MEMBERS && responseGiverTeam.equals(currentUserTeam)) {
-                return true;
-            } else if (type == FeedbackParticipantType.RECEIVER_TEAM_MEMBERS
-                    && responseRecipientTeam.equals(currentUserTeam)) {
-                return true;
-            } else if (type == FeedbackParticipantType.STUDENTS && roster.getStudentForEmail(userEmail) != null) {
-                return true;
+            switch (type) {
+            case INSTRUCTORS:
+                if (roster.getInstructorForEmail(userEmail) != null) {
+                    return true;
+                }
+                break;
+            case OWN_TEAM_MEMBERS:
+                if (responseGiverTeam.equals(currentUserTeam)) {
+                    return true;
+                }
+                break;
+            case RECEIVER:
+                if (userEmail.equals(response.recipient)) {
+                    return true;
+                }
+                break;
+            case RECEIVER_TEAM_MEMBERS:
+                if (responseRecipientTeam.equals(currentUserTeam)) {
+                    return true;
+                }
+                break;
+            case STUDENTS:
+                if (roster.getStudentForEmail(userEmail) != null) {
+                    return true;
+                }
+                break;
+            case GIVER:
+                if (userEmail.equals(response.giver)) {
+                    return true;
+                }
```

Comment:
Nope, from memory it can range from 0 to around 5 or 6 elements, although I'm not sure why this is being brought up.